how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb

```diff
@@ -151,24 +129,54 @@
     "    capture_hardware_metrics=True,\n",
     "    capture_stderr=True,\n",
     "    capture_stdout=True,\n",
-    "    source_files=\"hello_neptune.ipynb\",  # does not work in Colab\n",
+    "    source_files=\"hello_neptune.ipynb\",  # Doesn't work in Colab\n",
     "    tags=[\"quickstart\", \"notebook\"],\n",
-    "    dependencies=\"requirements.txt\",  # pass the path to the requirements.txt file\n",
+    "    dependencies=\"requirements.txt\",  # Passes the path to the requirements.txt file\n",
     ")"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "metadata": {
-    "tags": [
-     "comment"
-    ]
-   },
+   "metadata": {},
+   "source": [
+    "The metadata appears in the Neptune app. To view the newly created run in the app, use the link that appeared in the cell output."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
    "source": [
-    "**To open the run in the Neptune web app, click the link that appeared in the cell output.**\n",
+    "### Log metadata to your own project\n",
     "\n",
-    "You'll use the `run` object you just created to log metadata. You'll see the metadata appear in the app."
+    "You can also use your own project instead of a public one. Specify your credentials with the `project` and `api_token` parameters:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import neptune\n",
+    "from getpass import getpass\n",
+    "\n",
+    "run = neptune.init_run(\n",
+    "    project=\"workspace-name/project-name\",  # Replace with your workspace and project names\n",
+    "    api_token=getpass(\"h0dHBzOi8aHR0cHM4kl0jvYh3Kb8...6Lc\"), # Replace with your API token\n",
+    ")"
```

Comment:
The code cells should be identical apart from the credentials. Now the anonymous block has a bunch of extra args compared to the "own project" cell. Let's spare users the copy-pasting and make the cells equal.

And the `getpass()` argument has to be changed, as that string is what's shown next to the input box where you enter your token.

Reply:
Commented before push :D